### ISSUE TITLE
MISSING pre-processing / training scripts for code generation

### ISSUE THREAD

**Issue by ashwin296** [PRIME-RL/PRIME#58]
hey authors, thanks for open-sourcing the project.

in the arxiv paper, you mentioned that the outcome-based reward for code is set to be the unit test pass rate, and specified the processing for code problems (sec b.1).

however, this part looks to be missing in the current repository.

would you please consider releasing the relevant code for reproducing? or remove the code-orm-related content in the paper if that is overclaiming?

i raised the issue, because directly using pass rate without discretization is unlikely to provide good reward signals when training codellms in practice. 

i doubt if this part is really implemented, and would love to reproduce if you could provide more details. it looks to be non-trivial to have that in the verl pipeline.

thank you!

**Comment by cgq15** [comment]
Hi, could you please elaborate more on which part is missing?

We have implemented the code sandbox in [PRIME](https://github.com/PRIME-RL/PRIME/blob/main/training/verl/utils/reward_score/prime.py)

And it is also merged in [verl](https://github.com/volcengine/verl/blob/main/verl/workers/reward_manager/prime.py)

Our full training log is avaliable at [here](https://api.wandb.ai/links/thu_cgq-tsinghua-university/jsxzbm7h). You can fully reproduce the results.

**Comment by ashwin296** [comment]
Thank you!

To start with, the data pre-processing for code is missing. Is only math data used for training? Would you please double check on that?
